View Controllers/SearchViewController.m

```diff
@@ -120,7 +120,10 @@ - (void)fetchPlaces {
                         {
                             if(longitude >= myMinLongitude && longitude <= myMaxLongitude)
                             {
-                                [mutableArray addObject:place];
```

Comment:
looks good, but out of curiosity why are we using an array here and not a set?

Reply:
It's just in case we need to order the places by some property to display them on the tableview (like we did in the feed).